src/main/java/frc/robot/utilities/TOFSensorSimple.java

```diff
@@ -7,27 +7,42 @@
 import com.playingwithfusion.TimeOfFlight;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.Robot;
 
 /** Add your docs here. */
 public class TOFSensorSimple {
   private TimeOfFlight sensor;
   private double threshold;
+  private boolean isBeamBrokeSim;
 
   public TOFSensorSimple(int ID, double threshold) {
     sensor = new TimeOfFlight(ID);
```

Comment:
We should explicitly set the ranging mode to Short:
https://www.playingwithfusion.com/frc/2022/javadoc/com/playingwithfusion/TimeOfFlight.html#setRangingMode(com.playingwithfusion.TimeOfFlight.RangingMode,double)

This should be the default, but we want to make sure as it has better readings in bright light with a tradeoff of only 1.3 meters.  We can also explicitly set the time to as low as 24ms.

( Doesn't mention the default. )